app/index.js

```diff
@@ -258,16 +258,47 @@ module.exports = yeoman.generators.Base.extend({
   },
 
   installDeps: function () {
-    if (!this.options['skip-install']) {
-      this.on('end', function () {
-        this.options.plugins.push('grunt-karma');
-        if (this.options.coffee) {
-          this.options.plugins.push('coffee-script');
+
+    function save (where, things, cb) {
+      var sortedObject = require("sorted-object");
+      var writeFileAtomic = require("write-file-atomic");
+      var saveTarget = path.resolve(where, "package.json");
+
+      fs.readFile(saveTarget, function (er, data) {
+        try {
+          data = JSON.parse(data.toString("utf8"));
+        } catch (ex) {
+          er = ex;
         }
-        this.npmInstall(this.options.plugins, {
-          saveDev: true
+
+        if (er) {
+          return cb(null);
+        }
+
+        data["devDependencies"] = data["devDependencies"] || {};
+        Object.keys(things).forEach(function (t) {
+          data["devDependencies"][things[t]] = '*';
+        })
+
+        data["devDependencies"] = sortedObject(data["devDependencies"]);
+
+        data = JSON.stringify(data, null, 2) + "\n";
+        writeFileAtomic(saveTarget, data, function (er) {
```

Comment:
same as before, not the way to handle files in yeoman. See documentation reference.